graphbolt/build.sh

```diff
@@ -28,7 +28,7 @@ if ! [[ -z "${CUDAARCHS}" ]]; then
     TORCH_CUDA_ARCH_LIST=${LAST_ARCHITECTURE:0:-1}'.'${LAST_ARCHITECTURE: -1}
   fi
 fi
-CMAKE_FLAGS="-DCUDA_TOOLKIT_ROOT_DIR=$CUDA_TOOLKIT_ROOT_DIR -DUSE_CUDA=$USE_CUDA -DGPU_CACHE_BUILD_DIR=$BINDIR -DTORCH_CUDA_ARCH_LIST=$TORCH_CUDA_ARCH_LIST -DUSE_LIBURING=$USE_LIBURING"
```

Comment:
The build type should be specified externally: [script/build_dgl.sh](https://github.com/dmlc/dgl/blob/55cf23f9fb17814792f6de859e52ade4acf675d3/script/build_dgl.sh#L38)